Mittons.Fixtures/Docker/Gateways/IContainerGateway.cs

```diff
@@ -18,6 +18,8 @@ public interface IContainerGateway
 
         Task RemoveFileAsync(string containerId, string containerFilename, CancellationToken cancellationToken);
 
+        Task EmptyDirectoryAsync(string containerid, string directory, CancellationToken cancellationToken);
```

Comment:
The `containerId` parameter could follow the camel casing naming.

Reply:
Good catch, fixed all four instances of this!